trade-finance-manager-api/src/v1/controllers/auth/auth-provider-config.js

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+require('dotenv').config();
+
+const TENANT_SUBDOMAIN = process.env.AZURE_SSO_TENANT_SUBDOMAIN;
+const REDIRECT_URI = `${process.env.TFM_UI_URL}/${process.env.AZURE_SSO_REDIRECT_URI}`;
+const POST_LOGOUT_REDIRECT_URI = process.env.AZURE_SSO_POST_LOGOUT_URI;
```

Comment:
We could simplify the env vars here with destructuring and maybe creating a new `REDIRECT_URI` env var

Reply:
yes, this is inspired by Microsoft MSAL SSO example.